libqtile/widget/keyboardlayout.py

```diff
@@ -94,6 +95,8 @@ def set_keyboard(self, layout: str, options: str | None) -> None:
             logger.error("Can not change the keyboard layout:")
         except OSError:
             logger.error("Please, check that setxkbmap is available:")
+        else:
+            os.system("xmodmap $HOME/.Xmodmap")
```

Comment:
Perhaps we should test that the file exists first before calling it?

It might also be good to use the same `try` `check_output` approach used right before it, and catch and log any errors that arise.